explanation/service-endpoint-encryption.rst

```diff
@@ -41,3 +40,31 @@ Points of interest for this design:
    This updates endpoints with HTTPS and also distributes the CA
    certificates to all the application units across the cloud via
    Keystone.
+
+TLS Vault feature
+-----------------
+
+The TLS Vault feature is the method to use for deployments that use Vault as an intermediary CA.
```

Comment:
[* suggestion *] "TLS Vault is a useful feauture for deployments that use Vault as an intermediary CA."